### PR TITLE
chore(release): prepare release notes for Modules 21.10

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Release notes for:
* Centreon BAM 21.10.3
* Centreon MBI 21.10.3
* Centreon Autodiscovery 21.10.4

## Target version (i.e. version that this PR changes)

- [x] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -199,6 +235,14 @@ Release date: `February 18, 2022`
 
 ## Centreon Auto Discovery
 
+### 21.10.4
+
+Release date: `May 26, 2023`
+
+#### Bug fixes
+
+- [Configuration] For some host discovery providers where no credentials were actually required, the job creation wizard still demanded to enter credentials.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:244]:
```suggestion
- [Configuration] For some host discovery providers where no credentials were actually required, the job creation wizard no longer demands to enter credentials.
```

File: versioned_docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -200,6 +235,14 @@ Release date: `February 18, 2022`
 
 ## Centreon Auto Discovery
 
+### 21.10.4
+
+Release date: `May 26, 2023`
+
+#### Bug fixes
+
+- [Configuration] For some host discovery providers where no credentials were actually required, the job creation wizard still demanded to enter credentials.
```
Comment (smau-centreon) [inline versioned_docs/version-21.10/releases/centreon-commercial-extensions.md:244]:
```suggestion
- [Configuration] For some host discovery providers where no credentials were actually required, the job creation wizard no longer demands to enter credentials.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by Jordan91 — COMMENTED** [review]
Remove lines  in part MAP, no Map Engine in the 21.10

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -140,6 +140,19 @@ Release date: `December 16, 2021`
 
 ## Centreon BAM
 
+### 21.10.3
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:145]: change me before releasing

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -158,6 +171,14 @@ Release date: `December 7, 2021`
 
 ## Centreon MBI
 
+### 21.10.3
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:176]: change me before releasing

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -199,6 +220,14 @@ Release date: `February 18, 2022`
 
 ## Centreon Auto Discovery
 
+### 21.10.4
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:225]: change me before releasing

File: versioned_docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -141,6 +141,19 @@ Release date: `December 16, 2021`
 
 ## Centreon BAM
 
+### 21.10.3
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-21.10/releases/centreon-commercial-extensions.md:146]: change me before releasing

File: versioned_docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -159,6 +172,14 @@ Release date: `December 7, 2021`
 
 ## Centreon MBI
 
+### 21.10.3
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-21.10/releases/centreon-commercial-extensions.md:177]: change me before releasing

File: versioned_docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -200,6 +221,14 @@ Release date: `February 18, 2022`
 
 ## Centreon Auto Discovery
 
+### 21.10.4
+
+Release date: `May 26, 2023`
```
Comment (tuntoja) [inline versioned_docs/version-21.10/releases/centreon-commercial-extensions.md:226]: change me before releasing

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -140,6 +140,19 @@ Release date: `December 16, 2021`
...
+
+#### Bug fixes
+
+- [Banner] Fixed the display of empty skeleton.
+- [Install] Fixed an issue preventing the module installation when the 24x7 timeperiod parameter was not set.
+- [Monitoring] Fixed an issue in the BA monitoring redirection.
+- [Server] Fixed an issue in the KPI configuration export process to Remote Servers that caused the KPIs' states to be reset at each export of the configuration.
+- [Server] Fixed an issue that prevented the children Business Activities from being displayed on Remote Servers.
+- [Widget] Fixed an issue that made links to Business Activities broken on **BAM - Live Business Activity Tree**.
+
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:155]: Just checking.
From the board I see that https://centreon.atlassian.net/browse/MON-14460 and https://centreon.atlassian.net/browse/MON-12299  are embdeded but not reported here. Is it normal?
Not sure also to which issue this RN refers to - [Server] Fixed an issue in the KPI configuration export process to Remote Servers that caused the KPIs' states to be reset at each export of the configuration.

Comment (tuntoja) [reply i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md:156]: added https://centreon.atlassian.net/browse/MON-14460 that was missing
https://centreon.atlassian.net/browse/MON-12299 corresponds to L153

**Review by nmagniezCentreon — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]

**Pull request merged**